website/docs/d/iam_trusted_profile_link_list.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_link_list"
+description: |-
+  Get information about iam_trusted_profile_link_list
+subcategory: "Identity & Access Management (IAM)"
+---
+
+# ibm_iam_trusted_profile_link_list
+
+Retrieve list of IAM trusted profile link as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information, about trusted profile link, see [list link to a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#list-link)
+
+## Example Usage
+
+```terraform
+data "ibm_iam_trusted_profile_link_list" "iam_trusted_profile_link_list" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the iam_trusted_profile_link_list.
+* `links` - (Required, List) List of links to a trusted profile.
+  Nested scheme for **links**:
+	* `id` - (String) the unique identifier of the claim rule.
```

Comment:
Please re-arrage attributes in alphabetical order..
this is applicable to all other docs as well